atomics/unit_tests/kokkos_based/unsignedint.cpp

```diff
@@ -84,6 +84,12 @@ TEST(TEST_CATEGORY, atomic_operations_unsigned) {
     ASSERT_TRUE((TestAtomicOperations::AtomicOperationsTestIntegralType<unsigned int,
                                                                         TEST_EXECSPACE>(
         start, end - i, 12)));
+    ASSERT_TRUE((TestAtomicOperations::AtomicOperationsTestIntegralType<unsigned int,
```

Comment:
Found the culprit, having the tests called in the same batch as the others results in the wrapping functions receiving a signed typing during compilation. I'm making an ```AtomicOperationsTestUnsignedIntegralType``` set for the two wrapping unit tests just so avoid the bad typing.